doc/STREAMING_ITERABLES.md

```diff
@@ -7,11 +7,144 @@
 - [Iterable Streams](#iterable-streams)
   - [Table of Contents](#table-of-contents)
   - [Usage Guide](#usage-guide)
+    - [Wrapping Duplex Iterables](#wrapping-duplex-iterables)
+  - [Iterable Stream Types](#iterable-stream-types)
+    - [Source](#source)
+    - [Sink](#sink)
+    - [Transform](#transform)
+    - [Duplex](#duplex)
   - [Iterable Modules](#iterable-modules)
 
 ## Usage Guide
 
-**Coming Soon!**
+### Wrapping Duplex Iterables
```

Comment:
I'd express this in terms of what it enables you to do. Typically people know what they want to do but don't know how to do it.